docs/strategy-customization.md

```diff
@@ -334,17 +363,67 @@ Please always check the mode of operation to select the correct method to get da
 - `historic_ohlcv(pair, timeframe)` - Returns historical data stored on disk.
-- `get_pair_dataframe(pair, timeframe)` - This is a universal method, which returns either historical data (for backtesting) or cached live data (for the Dry-Run and Live-Run modes).
-- `orderbook(pair, maximum)` - Returns latest orderbook data for the pair, a dict with bids/asks with a total of `maximum` entries.
 - `market(pair)` - Returns market data for the pair: fees, limits, precisions, activity flag, etc. See [ccxt documentation](https://github.com/ccxt/ccxt/wiki/Manual#markets) for more details on Market data structure.
+- `ohlcv(pair, timeframe)` - Currently cached candle (OHLCV) data for the pair, returns DataFrame or empty DataFrame.
+- [`orderbook(pair, maximum)`](#orderbookpair-maximum) - Returns latest orderbook data for the pair, a dict with bids/asks with a total of `maximum` entries.
 - `runmode` - Property containing the current runmode.
 
-#### Example: fetch live / historical candle (OHLCV) data for the first informative pair
+#### Example Usages:
+
+#### *available_pairs*
+
+``` python
+if self.dp:
+    for pair, timeframe in self.dp.available_pairs:
+        print(f"available {pair}, {timeframe}")
+```
+
+#### *current_whitelist()*
+Imagine you've developed a strategy that trades the `1m` timeframe using signals generated from a `1d` timeframe on the top 10 volume pairs by volume. 
+
+The strategy might look something like this:
+
+*Scan through the top 10 pairs by volume using the `VolumePairList` every minute and use a 14 day ATR to buy and sell.*
+
+Due to the limited available data, it's impossible to resample our `1m` candles into daily candles for use in the 14 day ATR. Most exchanges limit us to just 500 candles which effectively gives us around 1/3 of a daily candle. We need 14 days at least!
+
+Since we can't resample our data we will have to use an informative pair; and since our whitelist will be dynamic we don't know which pair(s) to use.
+
+This is where calling `self.dp.current_whitelist()` comes in handy.
+
+```python
+class SampleStrategy(IStrategy):
+    # strategy init stuff...
+
+    ticker_interval = '1m'
+
+    # more strategy init stuff..
+
+    def informative_pairs(self):
+
+        # get access to all pairs available in whitelist. 
+        pairs = self.dp.current_whitelist()
+        # Assign tf to each pair so they can be downloaded and cached for strategy.
+        informative_pairs = [(pair, '1d') for pair in pairs]
+        return informative_pairs
+
+    def populate_indicators(self, dataframe, metadata):
+        # Get the informative pair
+        informative = self.dp.get_pair_dataframe(pair=metadata['pair'], timeframe='1d')
+        # Get the 14 day ATR.
+        atr = ta.ATR(informative, timeperiod=14)
+        # Assign the Daily atr to the 1 minute dataframe.
+        dataframe['daily_atr'] = atr
```

Comment:
I would argue that this might lead to unexpected results...

ATR is with frequency 1d - while `dataframe` is at 1m (or 5m...) frequency. 
So it'll initially lead to a ton of empty values - which will need to be ffilled to be comparable reliably.

it will also "throw away" all ATR values older than the 500 candles of the original timeframe - so for 1m timeframe, you'll only have one value (or no value) - depending if you're in the first 3rd of the day or not, as you'll only have 500 minutes of candles in the dataframe - and the ATR candle is dated to midnight.

Now using 5m timeframe might help with that ... but maybe the problem should be pointed out anyway ... just to be sure.

Reply:
Well it should be expected that anything that is averaged over a period of 14 days (with a daily sample rate) shouldn't change much in a period of 500 minutes. At most we would see two different values occurring around midnight...

The point is that we are able to get higher timeframe references to generate signals on lower timeframes. Alongside the dynamic accessing of our referenced pair. 

In my example I wasn't being too specific about the execution of the strategy or dataframe handling because it seemed arbitrary.  But I think you make a good point on using ffill if we were actually going to trade this example, but I didn't feel that would be appropriate for this part of the doc.

Reply:
well to be fair - i've only noticed this since this week someone copy/pasted a full documentation sample which had a misspelled import ... :laughing: 

would you have added this last week - i probably wouldn't have noticed / thought about it.

Reply:
haha yea, it's a bit of a trade off we have to make when writing howtos for anything.... 

Either assume the reader has *some* knowledge or be rigorously thorough -- the latter of which usually takes significant time upfront, but arguably could cut down time in the long run due to having to explain issues that arise from poor documentation. 

That being said, I think a healthy middle ground is best. If someone has trouble with this part of the doc, I can always explain further and update the doc *ad totum* *ad finitum*